setup/swc-installation-test-2.py

```diff
@@ -22,23 +22,9 @@
 dependency.
 """
 
-from __future__ import print_function  # for Python 2.6 compatibility
-
 import distutils.ccompiler as _distutils_ccompiler
 import fnmatch as _fnmatch
-try:  # Python 2.7 and 3.x
-    import importlib as _importlib
```

Comment:
Hm - we should still *run* when it's Python 2.7, just tell people "your Python is out of date".